protocol.md

```diff
@@ -217,7 +217,7 @@ The `Upload-Offset` header's value MUST be equal to the current offset of the
 resource. In order to achieve parallel upload the
 [Concatenation](#concatenation) extension MAY be used. If the offsets do not
 match, the Server MUST respond with the `409 Conflict` status without modifying
-the upload resource.
+the uploaded resource.
```

Comment:
Please keep the phrase "upload resource" here. We use it to refer to the endpoint that is created using POST requests.

Reply:
Aha, I suspected that before. Reverted!